.github/workflows/testing.yml

```diff
@@ -0,0 +1,34 @@
+name: CI Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_call:
+
+jobs:
+  tests:
+    name: "Python ${{ matrix.python-version }}"
+    runs-on: ubuntu-20.04
```

Comment:
ubuntu-latest?

Reply:
Ubuntu 20.04 is the last Ubuntu to have Python 3.8 available in `/usr`.